[2023-04-14T15:09:11.124+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: execute_24hr_data_scrape.id_2 scheduled__2023-04-14T14:53:00+00:00 [queued]>
[2023-04-14T15:09:11.132+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: execute_24hr_data_scrape.id_2 scheduled__2023-04-14T14:53:00+00:00 [queued]>
[2023-04-14T15:09:11.133+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T15:09:11.133+0000] {taskinstance.py:1283} INFO - Starting attempt 4 of 4
[2023-04-14T15:09:11.134+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T15:09:11.148+0000] {taskinstance.py:1303} INFO - Executing <Task(BashOperator): id_2> on 2023-04-14 14:53:00+00:00
[2023-04-14T15:09:11.154+0000] {standard_task_runner.py:55} INFO - Started process 2804 to run task
[2023-04-14T15:09:11.156+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'execute_24hr_data_scrape', 'id_2', 'scheduled__2023-04-14T14:53:00+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/exec_web_crawlers_dag.py', '--cfg-path', '/tmp/tmp8_1f490b']
[2023-04-14T15:09:11.157+0000] {standard_task_runner.py:83} INFO - Job 154: Subtask id_2
[2023-04-14T15:09:11.211+0000] {task_command.py:388} INFO - Running <TaskInstance: execute_24hr_data_scrape.id_2 scheduled__2023-04-14T14:53:00+00:00 [running]> on host 6f8cd9778af0
[2023-04-14T15:09:11.277+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=xiaochen
AIRFLOW_CTX_DAG_ID=execute_24hr_data_scrape
AIRFLOW_CTX_TASK_ID=id_2
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T14:53:00+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-14T14:53:00+00:00
[2023-04-14T15:09:11.278+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-04-14T15:09:11.279+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/src/news_comments_crawlers/crawlers/_EN_CNA.py --name CNA']
[2023-04-14T15:09:11.288+0000] {subprocess.py:86} INFO - Output:
[2023-04-14T15:09:13.200+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.201+0000] {subprocess.py:93} INFO - --DEBUG: __file__:     /opt/***/src/news_comments_crawlers/crawlers/config/Constants.py
[2023-04-14T15:09:13.202+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.202+0000] {subprocess.py:93} INFO - -- Session Object : Good to go.
[2023-04-14T15:09:13.203+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.204+0000] {subprocess.py:93} INFO - -- Folders: Good to go.
[2023-04-14T15:09:13.204+0000] {subprocess.py:93} INFO - []
[2023-04-14T15:09:13.204+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.205+0000] {subprocess.py:93} INFO - --DEBUG: Platform -  CNA
[2023-04-14T15:09:13.205+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.206+0000] {subprocess.py:93} INFO - --DEBUG: Start-time -  2023-04-13 15:09:13.199087
[2023-04-14T15:09:13.206+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.206+0000] {subprocess.py:93} INFO - --DEBUG: End-time -  2023-04-14 15:09:13.199087
[2023-04-14T15:09:13.207+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.207+0000] {subprocess.py:93} INFO - --DEBUG: Link Path - /opt/***/src/news_comments_crawlers/crawlers/out/2023-04-14/links/CNA_2023-04-14_15.txt
[2023-04-14T15:09:13.208+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.208+0000] {subprocess.py:93} INFO - --DEBUG: JSON Data Path - /opt/***/src/news_comments_crawlers/crawlers/out/2023-04-14/data/json/CNA_2023-04-14_15.json
[2023-04-14T15:09:13.209+0000] {subprocess.py:93} INFO - 
[2023-04-14T15:09:13.209+0000] {subprocess.py:93} INFO - --DEBUG: DF Data Path - /opt/***/src/news_comments_crawlers/crawlers/out/2023-04-14/data/df/CNA_2023-04-14_15.pickle
[2023-04-14T15:09:13.209+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2023-04-14T15:09:13.210+0000] {subprocess.py:93} INFO -   File "/opt/***/src/news_comments_crawlers/crawlers/_EN_CNA.py", line 197, in <module>
[2023-04-14T15:09:13.210+0000] {subprocess.py:93} INFO -     main()
[2023-04-14T15:09:13.210+0000] {subprocess.py:93} INFO -   File "/opt/***/src/news_comments_crawlers/crawlers/_EN_CNA.py", line 169, in main
[2023-04-14T15:09:13.211+0000] {subprocess.py:93} INFO -     link_list = c_scrape_links(USE_CACHE, LINK_FILEPATH, gather_urls)
[2023-04-14T15:09:13.211+0000] {subprocess.py:93} INFO - NameError: name 'c_scrape_links' is not defined
[2023-04-14T15:09:13.335+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-04-14T15:09:13.346+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-04-14T15:09:13.350+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=execute_24hr_data_scrape, task_id=id_2, execution_date=20230414T145300, start_date=20230414T150911, end_date=20230414T150913
[2023-04-14T15:09:13.374+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 154 for task id_2 (Bash command failed. The command returned a non-zero exit code 1.; 2804)
[2023-04-14T15:09:13.417+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-04-14T15:09:13.434+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
